### ISSUE TITLE
fixed header

### ISSUE THREAD

**Issue by marraia** [fredguile/ng2-md-datatable#53]
I need the grid to have the fixed header

**Comment by fredguile** [comment]
As you know a data-table is coming up officially with material2.0.0-beta.8, that shall support fixed header (albeit in upcoming versions). So the question about `ng2-md-datatable` is, do people would still want to use that lib? Feature-wise is it worth adding things to it? I should run a survey ;) Just react to this issue to share your opinion!

And as a disclaimer, that lib is "open-source" which means that it's not only me (or a company) who's solely responsible for it, but "everyone" that can/shall contribute to it.

**Comment by marraia** [comment]
Yes, people use it .. but it should have something to fix the header, even before updating the design material, there is nothing you can do to fix the header?

**Comment by fredguile** [comment]
I unfortunately don't have much time to do it right now, but I'm sure it boils down to:
- adding an input param "fixedHeader" to `MdDataTableHeaderComponent`
- tweaking the template and SCSS to get a fixed header (like in https://codepen.io/tjvantoll/pen/JEKIu for instance)

Feel motivated to try this out as a PR?
